web/src/app/selection/ChanWebhookSelect.tsx

```diff
@@ -0,0 +1,91 @@
+import { Add } from '@mui/icons-material'
+import { Chip, Grid, TextField } from '@mui/material'
+import React, { useState } from 'react'
+import { WebhookChip } from '../util/Chips'
+
+type ChanWebhookSelectProps = {
+  value: Array<string>
+  onChange: (value: Array<string>) => void
+}
+
+function isValidURL(str: string): boolean {
+  try {
+    // eslint-disable-next-line no-new
+    new URL(str)
+    return true
+  } catch {
+    return false
+  }
+}
+
+export const ChanWebhookSelect = (
+  props: ChanWebhookSelectProps,
+): JSX.Element => {
+  const [newURL, setNewURL] = useState<string>('')
+  const { value, onChange = () => {} } = props
+
+  const selected = props.value.map((v) => {
+    return (
+      <Grid item key={v}>
+        <WebhookChip
+          id={v}
+          onDelete={() => onChange(value.filter((f) => f !== v))}
+        />
+      </Grid>
+    )
+  })
+
+  return (
+    <Grid container spacing={1}>
+      <Grid container spacing={1}>
+        {props.value.length ? selected : ''}
+      </Grid>
+      <Grid container item>
+        <TextField
+          variant='outlined'
+          fullWidth
+          name='webhooks'
+          value={newURL}
+          onChange={(e) => {
+            setNewURL(e.target.value.trim())
+          }}
+          error={
+            (!isValidURL(newURL) && newURL.length > 0) ||
+            value.indexOf(newURL) > -1
```

Comment:
nit: use `.includes` instead
```suggestion
            value.includes(newURL)
```